assets/styles/variables/_specials.scss

```diff
@@ -315,6 +315,15 @@ $pullquote-margin-top: 1em !default;
 /// @type String
 $pullquote-margin-bottom: 1em !default;
 
+/// Width for pullquotes on elements with a class of `pullquote`.
+/// @type String | Map
+/// @since 1.0.0
+$pullquote-width: (
+    epub: 65%, 
```

Comment:
These should default to `auto`, not `65%`; otherwise it would change all the existing layouts.